packages/@okta/vuepress-site/docs/api/resources/zones/index.md

```diff
@@ -2,23 +2,160 @@
+## Zone Model
+
+### Base Network Zone Properties
+
+The Base Network Zone object defines the base properties which concrete Network Zone types (IP and Dynamic) inherit.
+
+All Network Zone objects define the following attributes:
+
+| Field Name     | Description                                                                                 | Data Type                                     | Required        | Max Length    |
+| :------------- | :------------------------------------------------------------------------------------------ | :-------------------------------------------- | :-------------- | :------------ |
+| type           | Type of zone is one of the following: `IP`, `DYNAMIC`                                              | String                                        | Yes             | N/A           |
+| id             | Unique identifier for this zone                                                             | String                                        | No (Assigned)   | N/A           |
+| name           | Unique name for this zone                                                                   | String                                        | Yes             | 128 (chars)   |
+| system           | Indicates if this is a system network zone. For admin-created zones, this is always `false`                                                                 | boolean                                        | No  (Assigned)           | N/A   |
+
+### IP Zone Properties
+
+The follow attributes are defined by IP Zone objects:
+
+| Field Name     | Description                                                                                 | Data Type                                     | Required        | Max Length    |
+| :------------- | :------------------------------------------------------------------------------------------ | :-------------------------------------------- | :-------------- | :------------ |
+| gateways       | IP addresses (range or CIDR form) of this zone                                              | Array of [Address Objects](#address-object)   | No              | 150 (entries) |
+| proxies        | IP addresses (range or CIDR form) allowed to forward request from gateway addresses above   | Array of [Address Objects](#address-object)   | No              | 150 (entries) |
+
+#### Address Object
+
+Each address object specifies a set of IP addresses, expressed using either range or CIDR form.
+
+| Field Name  | Description                                                | Data Type   | Required |
+| :---------- | :--------------------------------------------------------- | :---------- | :------- |
+| type        | Format of the value: either CIDR or RANGE                 | String      | Yes       |
+| value       | Value in CIDR/RANGE form depending on the type specified   | String      | Yes       |
```

Comment:
Please be consistent with capitalization: `RANGE` vs `Range`.

Reply:
RANGE is the string value used to define a type. I believe the use of RANGE under the `type` row is correct whereas the use of RANGE in the `value` row is incorrect